db/migrate/20200525084440_add_latest_flag_to_timeline_event_owners_table.rb

```diff
@@ -0,0 +1,19 @@
+class AddLatestFlagToTimelineEventOwnersTable < ActiveRecord::Migration[6.0]
+  class TimelineEvent < ApplicationRecord
+    has_many :timeline_event_owners
+  end
+
+  class TimelineEventOwner < ApplicationRecord
+    belongs_to :timeline_event
+  end
+
+  def change
+    add_column :timeline_event_owners, :latest, :boolean, default: false
+
+    TimelineEventOwner.reset_column_information
+
+    TimelineEvent.where(latest: true).map do |submission|
+      submission.timeline_event_owners.update_all(latest: true)
+    end
```

Comment:
This doesn't seem  to do anything to fix the issues with multiple submissions being marked as latest, which is present in the DB right now. 🤔 

Some kind of re-computation will need to be performed for all team submissions.